Style/Frontend/index.md

```diff
@@ -0,0 +1,1451 @@
+  export function square(x) {
+    return x**2;
+  }
+
+  // good, but consider the next point
+  const square = (x) => {
+    return x**2;
+  }
+  ```
+
+- Implicitly return where possible
+
+  ```jsx
+  // best
+  const square = (x) => x**2;
+  ```
+
+- Always include parentheses around arguments for clarity and consistency.
+
+  > This minimizes diff churn when adding or removing arguments
+
+  ```jsx
+  // bad
+  const square = x => x**2;
+
+  // good
+  const square = (x) => x**2;
+  ```
+
+- Don't provide default arguments
+
+  > Consider the following real WOLF example. This function takes in a number and an optional object that contains the specific configuration of how that number should be formatted. It would be incredibly unwieldy if every time you called it you gave it every single option, even if you don't actually need it.
+
+  ```jsx
+  export function formatNumber(
+    number,
+    {
+      decimalPlace = 2,
+      abbreviateK = false,
+      smartAbbreviate = true,
+      includeDollar = true,
+      includeMinus = true,
+      includePlus = false,
+      includePercent = false,
+      addCommas = true,
+      min = null,
+      toFixed = true,
+    } = {},
+  ) { ... }
+  
+  // bad
+  formatNumber("1000.00", {decimalPlace: 2, abbreviateK: false});
+     
+  // good
+  // default arguments are automatically applied
+  formatNumber("1000.00");
+  
+  // only provide the configurations where it changes
+  formatNumber("1000.00", {includePlus: true});
+  ```
+
+## Blocks
+
+- Use braces with all multiline blocks
+
+  ```jsx
+  // bad
+  if (test)
+    return false;
+
+  // good
+  if (test) return false;
+
+  // good
+  if (test) {
+    return false;
+  }
+  ```
+
+- If an `if` block always executes a `return` statement, the subsequent `else` block is unnecessary. A `return` in an `else if` block following an `if` block that contains a `return` can be separated into multiple `if` blocks.
+
+  ```jsx
+  // bad
+  const foo = () => {
+    if (x) {
+      return x;
+    } else {
+      return y;
+    }
+  }
+  
+  // bad
+  const foo = () => {
+    if (x) {
+      return x;
+    } else if (y) {
+      return y;
+    }
+  }
+  
+  // bad
+  const foo = () => {
+    if (x) {
+      return x;
+    } else {
+      if (y) {
+        return y;
+      }
+    }
+  }
+  
+  // good
+  const foo = () => {
+    if (x) {
+      return x;
+    }
+  
+    return y;
+  }
+  
+  // good
+  const foo = () => {
+    if (x) {
+      return x;
+    }
+  
+    if (y) {
+      return y;
+    }
+  }
+  
+  // best
+  const foo = () => {
+    return x ?? y;
+  }
+  ```
+
+## Control Statements
+
+- Don't use selection operators in place of control statements for running functions
+
+  ```jsx
+  // bad
+  !isRunning && startRunning();
+  
+  // good
+  if (!isRunning) {
+    startRunning();
+  }
+  
+  // we do this a lot in JSX, fine to do for literals and variables
+  <View>
+    <Text>{hasProperty && 'Property detected'}</Text>
```

Comment:
I'd fix this example so it's `{hasProperty && <Text>Property detected</Text>}` since if hasPropperty is false, it might render the word false? Or some other unpredictable behavior.